tool/publish.sh

```diff
@@ -1,11 +1,13 @@
+#!/bin/bash
+
 set -e
 
 if [ ! -f pubspec.yaml ]; then
     echo "Error: No pubspec.yml in working directory."
     exit
 fi
 
-cp ../../LICENSE LICENSE
-cp ../../README.md README.md
+cp ../LICENSE LICENSE
```

Comment:
where do we still have a license that lives in a different directory than the package?